server/Helper.js

```diff
@@ -0,0 +1,222 @@
+import Validator from 'validator';
+import pool from './DatabaseConnection';
+import jwt from 'jsonwebtoken';
+/*
+*@ helper functions
+* @trimeWhiteSpace given an object, it removes the whitespaces in the values of the object
+* @validateKey give an object and array of key you expect the object to have, it validates 
+* if the obejct contains the key
+*@validateInput ensure that the inputs are valid 
+*/
+class Helper {
+    constructor(){
+        this.executeQuery = this.executeQuery.bind(this);
+        this.connectToDb = this.connectToDb.bind(this);
+        // this.assignToken = this.assignToken.bind(this);
+    }
+    // trimWhiteSpace(obj){
+    //     if(typeof obj !== "undefined" && obj !== '' && typeof obj === 'object' && typeof obj.length === "undefined"){
+           
+    //         Object.keys(obj).forEach(function(key){ 
```

Comment:
Why do you have commented out code??

Reply:
The functionality is not been used yet, so I don't want it to affect my code coverage